nni/retiarii/oneshot/pytorch/strategy.py

```diff
@@ -91,8 +89,11 @@ class DARTS(OneShotStrategy):
     def __init__(self, **kwargs):
         super().__init__(DartsLightningModule, **kwargs)
 
-    def _get_dataloader(self, train_dataloader, val_dataloaders):
-        return InterleavedTrainValDataLoader(train_dataloader, val_dataloaders)
+    def preprocess_dataloader(self, train_dataloaders, val_dataloaders):
+        return {
+            'train': train_dataloaders,
+            'val': val_dataloaders
+        }, None
```

Comment:
the return signature is changed from the base class?

Reply:
I put both train and val dataloaders into the train dataloader, and we don't need a val dataloader.

Reply:
I know, but the first returned value is a dict, not a train dataloader?